src/java/apiview-java-processor/pom.xml

```diff
@@ -53,6 +53,11 @@
       <version>4.0.0</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>commons-lang</groupId>
+          <artifactId>commons-lang</artifactId>
+          <version>2.6</version>
+      </dependency>
```

Comment:
Ideally we wouldn't add an extra dependency, but it isn't the end of the world if we do.